test/v200/utils/common/command_test_utils.go

```diff
@@ -0,0 +1,212 @@
+package common
+
+import (
+	"fmt"
+
+	schema "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+)
+
+// commandAdded adds a new command to the test schema data and notifies the follower
+func (testDevFile *TestDevfile) commandAdded(command schema.Command) {
+	LogInfoMessage(fmt.Sprintf("command added Id: %s", command.Id))
+	testDevFile.SchemaDevFile.Commands = append(testDevFile.SchemaDevFile.Commands, command)
+	if testDevFile.Follower != nil {
+		testDevFile.Follower.AddCommand(command)
+	}
+}
+
+// commandUpdated and notifies the follower of the command which has been updated
+func (testDevFile *TestDevfile) commandUpdated(command schema.Command) {
+	LogInfoMessage(fmt.Sprintf("command updated Id: %s", command.Id))
+	if testDevFile.Follower != nil {
+		testDevFile.Follower.UpdateCommand(command)
+	}
+}
+
+// addEnv creates and returns a specifed number of env attributes in a schema structure
+func addEnv(numEnv int) []schema.EnvVar {
+	commandEnvs := make([]schema.EnvVar, numEnv)
+	for i := 0; i < numEnv; i++ {
```

Comment:
same as https://github.com/devfile/api/pull/348/files#r579477717
and for other for loops

Reply:
Also in this case I am adding entries to a map, not iterating through it, so I don't think range works.

Reply:
thanks for the explanation, I did not check the loop clearer